Share The Load/Screens/ViewController.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  ViewController.swift
+//  Share The Load
+//
+//  Created by Christopher Hefferman on 10/01/2021.
+//
+
+import UIKit
+import Firebase
+import RealmSwift
+
+class ViewController: UIViewController {
+    
+    private let getRealmQuestionsButton = UIButton()
+    private let removeFromRealmButton = UIButton()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        fetchQuiz()
+        createGetRealmQuestionsButton()
+        createRemoveButton()
+    }
+    
+    
+    private func createGetRealmQuestionsButton() {
+        view.addSubview(getRealmQuestionsButton)
+        getRealmQuestionsButton.translatesAutoresizingMaskIntoConstraints = false
+        getRealmQuestionsButton.setTitle("Get Questions From Realm", for: .normal)
+        getRealmQuestionsButton.frame = view.frame
+        getRealmQuestionsButton.backgroundColor = .systemBlue
+        getRealmQuestionsButton.layer.cornerRadius = 20
+        getRealmQuestionsButton.addTarget(self, action: #selector(buttonTapped), for: .touchUpInside)
+        
+        NSLayoutConstraint.activate([
+            getRealmQuestionsButton.centerYAnchor.constraint(equalTo: view.centerYAnchor),
+            getRealmQuestionsButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            getRealmQuestionsButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
+            getRealmQuestionsButton.heightAnchor.constraint(equalToConstant: 50)
+        ])
+    }
+    
+    
+    private func createRemoveButton() {
+        view.addSubview(removeFromRealmButton)
+        removeFromRealmButton.translatesAutoresizingMaskIntoConstraints = false
+        removeFromRealmButton.setTitle("Remove Question", for: .normal)
+        removeFromRealmButton.frame = view.frame
+        removeFromRealmButton.backgroundColor = .systemRed
+        removeFromRealmButton.layer.cornerRadius = 20
+        removeFromRealmButton.addTarget(self, action: #selector(removeButtonTapped), for: .touchUpInside)
+        
+        NSLayoutConstraint.activate([
+            removeFromRealmButton.topAnchor.constraint(equalTo: getRealmQuestionsButton.bottomAnchor, constant: 20),
+            removeFromRealmButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            removeFromRealmButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
+            removeFromRealmButton.heightAnchor.constraint(equalToConstant: 50)
+        ])
+    }
+    
+    
+    @objc func buttonTapped() {
```

Comment:
Which button?

Reply:
Yep - I see it. Thanks - changed to: 'getRealmQuestionsButtonTapped'.